textdb/textdb-dataflow/pom.xml

```diff
@@ -59,13 +59,16 @@
             <version>3.6.0</version>
             <classifier>models</classifier>
         </dependency>
-
-
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
             <version>1.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
So, if you notice the latest commit, you will realize that the `org.json` module isn't even needed. I do not know why that module was even introduced in the first place.